packages/next/src/client/components/router-reducer/fetch-server-response.ts

```diff
@@ -153,14 +160,12 @@ export async function fetchServerResponse(
     }
 
     // Handle the `fetch` readable stream that can be unwrapped by `React.use`.
-    const [buildId, flightData]: NextFlightResponse = await createFromFetch(
+    const [newBuildId, flightData]: NextFlightResponse = await createFromFetch(
```

Comment:
Perhaps `buildIdFromResponse`, or `incomingBuildId`? To me `newBuildId` suggests that it's not the same ID as previously, which isn't necessarily the case.